### PR TITLE
[Translation it] Fundamentals > Payments > Android Pay

### PR DESCRIPTION
**Pull request by lucab85**
Italian translation of Fundamentals > Payments > Android Pay.

Please review by @carlaofarrell

[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/it/review/485)

### CODE REVIEW

File: src/content/it/fundamentals/payments/android-pay.md [hunk 1]
```diff
@@ -0,0 +1,510 @@
+project_path: /web/fundamentals/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Android Pay consente acquisti online semplici e sicuri ed elimina
```
Comment (petele) [inline src/content/it/fundamentals/payments/android-pay.md]: description should be on a single line, no line-breaks in the description.

Comment (sotayamashita) [reply src/content/it/fundamentals/payments/android-pay.md]: @lucab85 I have created a pull request to fix the issue @petele mention ↑. Please check it out. https://github.com/lucab85/WebFundamentals/pull/1

File: src/content/it/fundamentals/payments/android-pay.md [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+**Sicuro:** Android Pay funziona memorizzando in modo sicuro un numero di
+account virtuale associato all'account di pagamento di un utente. Ciò consente
+acquisti online senza che l'utente debba inviare il proprio numero reale di
+carta di credito o di debito. Android Pay crittografa ogni transazione di
+pagamento, mantenendo al sicuro i dati dell'utente.
+
+**Supporto:** Android Pay è supportato in un numero crescente di paesi e dalla
+maggior parte delle principali circuiti di carte di credito e banche ed è
+disponibile su tutti i telefoni Android con KitKat e versioni successive. Fare
+riferimento a questa [pagina del centro
```
Comment (carlaofarrell) [inline src/content/it/fundamentals/payments/android-pay.md]: consulta questa [pagina del centro

File: src/content/it/fundamentals/payments/android-pay.md [hunk 3]
```diff
@@ -0,0 +1,506 @@
...
+della risposta sono utili.
+- Esamina le [norme relative ai
+contenuti](https://support.google.com/payments/merchant/answer/75724?payments_to_biz=&rd=1)
+dettagliate di Android Pay per assicurarti che i tuoi specifici prodotti o
+servizi siano supportati.
+
+### Impostazione dell'ambiente
+
+- Assicurati di avere l'app Android Pay installata sul tuo dispositivo. È
+necessario essere in uno dei paesi supportati per installarlo. Controlla su
```
Comment (carlaofarrell) [inline src/content/it/fundamentals/payments/android-pay.md]: necessario essere in uno dei paesi supportati per installarla. Controlla su

File: src/content/it/fundamentals/payments/android-pay.md [hunk 4]
```diff
@@ -0,0 +1,506 @@
...
+  }
+];
+```
+
+Per utilizzare Android Pay con l'approccio token gateway, aggiungi un oggetto
+JSON che contenga i seguenti parametri per l'esempio precedente.
+
+- `supportedMethods: [ 'https://android.com/pay' ]`: indica che si tratta di un
+metodo di pagamento che utilizza Android Pay.
+- `data` : si tratta di valori specifici per Android Pay che non sono ancora
```
Comment (carlaofarrell) [inline src/content/it/fundamentals/payments/android-pay.md]: - `data` : indica valori specifici per Android Pay che non sono ancora

**Comment by lucab85** [comment]
Integrated @carlaofarrell 's review and updated `wf_updated_on`.

**Pull request merged**